StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp9/OrderingRules/SA1212CSharp9UnitTests.cs

```diff
@@ -3,9 +3,91 @@
 
 namespace StyleCop.Analyzers.Test.CSharp9.OrderingRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
```

Comment:
Needed for `ReferenceAssemblies`.

```suggestion
    using Microsoft.CodeAnalysis.Testing;
```